src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import {Filter} from "../../types/Filter";
+
+type Props = {
+  todos: Todo[];
+  handleFilterChange: (filter: string) => void;
+  selectedOption: string;
+};
+export const Footer: React.FC<Props> = ({
+  todos,
+  handleFilterChange,
+  selectedOption,
+}) => {
+  const todoCounter = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todoCounter} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link',
+            { selected: selectedOption === 'All' })}
+          data-cy="FilterLinkAll"
+          onClick={() => handleFilterChange(Filter.All)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link',
+            { selected: selectedOption === 'Active' })}
```

Comment:
why aren't u using your enum with values everywhere in the project?
```suggestion
            { selected: selectedOption === Filter.Active })}
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**